src/components/ComputeModal.js

```diff
@@ -1647,25 +1650,65 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     ])
   }
 
-  const renderPersistentDiskSection = () => {
+  const renderPersistentDiskSection = diskExists => {
+    const gridStyle = { display: 'grid', gridGap: '1.3rem', alignItems: 'center', marginTop: '1rem' }
+
     return div({ style: { ...computeStyles.whiteBoxContainer, marginTop: '1rem' } }, [
       h(IdContainer, [
         id => h(div, { style: { display: 'flex', flexDirection: 'column' } }, [
-          label({ htmlFor: id, style: computeStyles.label }, ['Persistent disk size (GB)']),
+          label({ htmlFor: id, style: computeStyles.label }, ['Persistent disk']),
           div({ style: { marginTop: '0.5rem' } }, [
             'Persistent disks store analysis data. ',
             h(Link, { onClick: handleLearnMoreAboutPersistentDisk }, ['Learn more about persistent disks and where your disk is mounted.'])
           ]),
-          h(NumberInput, {
-            id,
-            min: 10,
-            max: 64000,
-            isClearable: false,
-            onlyInteger: true,
-            value: computeConfig.selectedPersistentDiskSize,
-            style: { marginTop: '0.5rem', width: '5rem' },
-            onChange: updateComputeConfig('selectedPersistentDiskSize')
-          })
+          div({ style: { ...gridStyle, gridGap: '1rem', gridTemplateColumns: '15rem 4.5rem', marginTop: '0.75rem' } }, [
+            diskExists ?
+              h(TooltipTrigger, { content: ['Disk type can only be selected at creation time.'], side: 'bottom' }, [
+                h(div, [
+                  label({ htmlFor: id, style: computeStyles.label }, ['Disk type']),
+                  div({ style: { marginTop: '0.5rem' } }, [
+                    h(Select, {
+                      id,
+                      value: computeConfig.selectedPersistentDiskType,
+                      isDisabled: diskExists || false,
+                      onChange: ({ value }) => updateComputeConfig('selectedPersistentDiskType', value),
+                      options: [
+                        { label: pdTypes.standard.displayName, value: pdTypes.standard.label },
+                        { label: pdTypes.ssd.displayName, value: pdTypes.ssd.label }
+                      ]
+                    })
+                  ])
+                ])
```

Comment:
It may be good to pull this snippet out to a method like 

```javascript
const renderPersistentDiskType = …
```

since it's duplicated right below.